src/test/kotlin/org/malachite/estella/queue/TaskResultTest.kt

```diff
@@ -80,13 +80,25 @@ class TaskResultTest : BaseIntegration() {
     @Order(1)
     fun `task result consuming`() {
         expectThat(taskStage.tasksResult.size).isEqualTo(0)
+        val code = SerialClob("xd".toCharArray())
+        val results =  SerialBlob("xd".toByteArray())
 
-        val taskResult = TaskResult(null, SerialBlob("xd".toByteArray()), SerialClob("xd".toCharArray()), null, null, task, taskStage)
+        val taskResult = TaskResult(null, results, code, null, null, task, taskStage)
         publish(taskResult)
 
         eventually {
             taskStage = taskStageRepository.findById(taskStage.id!!).get()
             expectThat(taskStage.tasksResult.size).isEqualTo(1)
+            expectThat(code.characterStream.readText()).isEqualTo("xd")
+            expectThat(Base64.getDecoder().decode(taskStage.tasksResult[0].results.binaryStream.readAllBytes())).isEqualTo("xd".toByteArray())
```

Comment:
To w środku decode to obowiązkowo do jakiejś intermediate zmiennej wcześnie bo źle się patrzy na to